phac-nml/irida-next
# Project Attachments: Destroy

## What does this PR do and why?
This PR adds file deletion to project attachments.

## Screenshots or screen recordings
[Screencast from 2024-09-17 02:21:17 PM.webm](https://github.com/user-attachments/assets/40f5a99e-ac00-423a-942c-cc6561d4c48c)

## How to set up and validate locally
1. Navigate to a project `Files` page.
2. Upload files if needed (including paired end files)
3. Delete the files using the `Delete` link within each row (paired end files should both delete with a single action)
4. Ensure the table updates and no longer includes the deleted file(s)

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
